pkg/api/response/response.go

```diff
@@ -292,6 +292,11 @@ func ErrOrFallback(status int, message string, err error) *NormalResponse {
 		return Err(err)
 	}
 
+	grafanaBaseErr := errutil.Base{}
+	if errors.As(err, &grafanaBaseErr) {
+		return Err(grafanaBaseErr.Errorf("%w", grafanaBaseErr))
+	}
```

Comment:
this allows to use `ErrOrFallback` with `errutil.Base` which is helpful for cases like in  [client.go](https://github.com/grafana/grafana/pull/74928/files#diff-f68ce913764de9b2ec9c118f29476e2aeec3981d7561124e48659e5a8c2c6f78L69)